<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Id_canvas">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>14</y>
     <width>471</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_black">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_red">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_blue">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
